wgpu-hal/src/gles/adapter.rs

```diff
@@ -1136,7 +1136,7 @@ impl crate::Adapter<super::Api> for super::Adapter {
             Some(crate::SurfaceCapabilities {
                 formats,
                 present_modes: if cfg!(windows) {
-                    vec![wgt::PresentMode::Fifo, wgt::PresentMode::Mailbox]
+                    vec![wgt::PresentMode::Fifo, wgt::PresentMode::Immediate]
```

Comment:
I asked about this on the initial PR - I'm not sure that you can tear with wgl and the windows compositor (cc @Zoxc?)

Reply:
Tearing can be observed on my windows laptop.